src/server/components/lists/helpers.ts

```diff
@@ -160,3 +164,21 @@ export function createListSearchBaseLink(serviceType: string): string {
   const protocol = isLocalHost ? "http" : "https";
   return `${protocol}://${SERVICE_DOMAIN}${listsRoutes.finder}?serviceType=${serviceType}`;
 }
+
+export function createFormRunnerReturningUserLink(serviceType: string): string {
+  if (serviceType === undefined) {
```

Comment:
a neat trick I use is 
```
  if(serviceType ?? true) 
```

(if serviceType is null or undefined, force this block of code to run)... I suspect the es/ts rules won't like this though

Reply:
yep, the producing warning is `ESLint: Unexpected string value in conditional. An explicit empty string check is required.(@typescript-eslint/strict-boolean-expressions)` with the recommended solution to `check for empty string (value !== "")`.  Is it generally bad practice to override default TS validation in eslint?

Reply:
TS is only a superset of js. I think the TS rules are what Microsoft `think` are best practice, but they're too strict and work against modern js imo

Reply:
As discussed, looks like this one can't be implemented in this way so keeping as `serviceType != undefined`